src/layouts/MainLayout/BottomNav.css

```diff
@@ -1,9 +1,13 @@
-.MuiBottomNavigationAction-root.Mui-selected {
+/* .MuiBottomNavigationAction-root.Mui-selected {
```

Comment:
Hey, @faisaljamil25 I think you should not keep any commented codes.

Reply:
Hey, @sakshi-choudhary actually I was waiting for mentors' confirmation whether I should remove the bottom border or not. That's why I commented for now. I will either remove it if I get confirmation or revert back.

Reply:
okay